src/applications/vaos/utils/appointment.jsx

```diff
@@ -66,6 +66,12 @@ export function titleCase(str) {
     .join(' ');
 }
 
+export function sentenceCase(str) {
+  return `${str.charAt(0).toUpperCase()}${str
```

Comment:
I think this will change `MOVE! weight management program` to `Move! weight management program`. We might need to handle words that are all caps.